### ISSUE TITLE
frontend: create admin override for phases blocked by insufficient commits

### ISSUE THREAD

**Issue by pawlh** [softwareconstruction240/autograder#47]

**Comment by frozenfrank** [comment]
@pawlh I think that recent discussions have indicated that we aren't blocking submissions because of a minimum commit count. Therefore, should we close this issue as "not planned?"

**Comment by pawlh** [comment]
> [@pawlh](https://github.com/pawlh) I think that recent discussions have indicated that we aren't blocking submissions because of a minimum commit count. Therefore, should we close this issue as "not planned?"

Yes, this is superseded by #160
